packages/canary-react/README.md

```diff
@@ -58,6 +58,24 @@ The `@ukic/canary-react` package will need to be transformed before you can use
 
 Add a `transformIgnorePatterns` field with the value `["/node_modules/(?!@ukic/canary-react)"]` to your Jest config.
 
+## Typescript
+
+If your project is making use of Typescript, you can import the custom types used in the ICDS component library. For details on the types implemented in each component, refer to the Props table on the Code page for each Component on the [Design System Guidance site](https://design.sis.gov.uk/components/select/code)
+
+Types can be imported from the @ukic/canary-web-components package as per the below example. NOTE: you do not have to install this package into your project, it is already installed as a dependency of @ukic/canary-react. 
```

Comment:
For notes, use the markdown syntax for it.

https://github.com/orgs/community/discussions/16925#discussion-4085374